lib/google_maps_geocoder/google_maps_geocoder.rb

```diff
@@ -183,11 +183,11 @@ def parse_formatted_street_address
   end
 
   def parse_lat
-    @json['results'][0]['geometry']['location']['lat']
+    @json[:results][0][:geometry][:location][:lat]
```

Comment:
Minor: Another refactoring would be to use Hash#dig and Array#dig for these long ones.